### PR TITLE
Add electron-compile

### PR DESCRIPTION
**Pull request by paulcbetts**
Like it says on the tin

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Some good apps written with Electron.
 - [electron-gh-release](https://github.com/jenslind/electron-gh-releases) - Auto-update by releasing on GitHub.
 - [electron-updater](https://github.com/evolvelabs/electron-updater) - Auto-updater leveraging npm to deploy updates.
 - [electron-download](https://github.com/maxogden/electron-download) - Download the Electron release zip from GitHub.
+- [electron-compile](https://github.com/paulcbetts/electron-compile) - Compile JS and CSS automatically in Electron apps
```
Comment (shinnn) [inline readme.md]: https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md

> Start the description with a capital and end with a full stop/period.

File: readme.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ Some good apps written with Electron.
 - [electron-gh-release](https://github.com/jenslind/electron-gh-releases) - Auto-update by releasing on GitHub.
 - [electron-updater](https://github.com/evolvelabs/electron-updater) - Auto-updater leveraging npm to deploy updates.
 - [electron-download](https://github.com/maxogden/electron-download) - Download the Electron release zip from GitHub.
+- [electron-compile](https://github.com/paulcbetts/electron-compile) - Compile JS and CSS automatically in Electron apps.
```
Comment (sindresorhus) [inline readme.md]: Confusing description as it's not really compiling JS/CSS, but rather precompiling down to those. Also `Electron` is moot as it's implied.

**Comment by sindresorhus** [comment]
Glad to see this ready :)

File: readme.md (hunk 2 above)
Comment (sindresorhus) [inline readme.md]: @paulcbetts

Comment (paulcbetts) [inline readme.md]: hwhoops, will fix this up once I'm at a Better Computer(tm)

File: readme.md [hunk 3]
```diff
@@ -72,6 +72,7 @@ Some good apps written with Electron.
 - [electron-gh-release](https://github.com/jenslind/electron-gh-releases) - Auto-update by releasing on GitHub.
 - [electron-updater](https://github.com/evolvelabs/electron-updater) - Auto-updater leveraging npm to deploy updates.
 - [electron-download](https://github.com/maxogden/electron-download) - Download the Electron release zip from GitHub.
+- [electron-compile](https://github.com/paulcbetts/electron-compile) - Use ES6, Coffeescript, and LESS/SCSS directly in Electron without a precompilation step.
```
Comment (paulcbetts) [inline readme.md]: @sindresorhus Thoughts?

Comment (sindresorhus) [inline readme.md]: > Use ES2015, CoffeeScript, LESS, SCSS in your app without a precompilation step.

**Comment by paulcbetts** [comment]
Anything else?

**Comment by sindresorhus** [comment]
Nope, just don't get notification on new commits.

**Comment by paulcbetts** [comment]
@sindresorhus Thanks! :sparkling_heart:

**Pull request closed without merging**